cmd/platform/role/update.go

```diff
@@ -0,0 +1,62 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package cmdrole
+
+import (
+	"fmt"
+
+	"github.com/elastic/cloud-sdk-go/pkg/api/platformapi/roleapi"
+	"github.com/elastic/cloud-sdk-go/pkg/models"
+	"github.com/elastic/cloud-sdk-go/pkg/util/cmdutil"
+	"github.com/spf13/cobra"
+
+	"github.com/elastic/ecctl/pkg/ecctl"
+)
+
+var updateCmd = &cobra.Command{
+	Use:     "update <role>",
+	Short:   "Updates an existing platform role",
+	PreRunE: cobra.MinimumNArgs(1),
```

Comment:
We've been using https://pkg.go.dev/github.com/spf13/cobra#ExactArgs for the new commands too, since it provides a little bit better expectations, if >1 args are passed, then they're ignored rather than an error returned.
```suggestion
	PreRunE: cobra.ExactArgs(1),
```